eve/main.yml

```diff
@@ -224,6 +224,17 @@ stages:
           command: bash cypress.sh
           workdir: build/ui
           haltOnFailure: true
+      - ShellCommand:
+          name: Create temporary folder to upload Cypress result
+          command: >
+            mkdir -p tmp/cypress &&
+            cp -r cypress/videos tmp/cypress
```

Comment:
Two remarks:

- The temp dir you created could actually be a more general `build/artifacts` directory in which all artifacts related to that step are stored (since we can't actually control how to push artifacts...). But it's a "nice-to-have", feel free to ignore
- You only include `cypress/videos`, while your PR mentions `cypress/screenshots` as well. I get that there are none generated right now, but adding it now would avoid surprises in the future IMO.

Reply:
I'm not sure about your first remark.
I have made the change for the second one.